clients/fides-js/src/integrations/gtm.ts

```diff
@@ -0,0 +1,19 @@
+declare global {
```

Comment:
The rest of the `src/***` files in `fides-js` are unchanged copies from the `fides-consent` package (gtm, meta, shopify, consent-config, etc.)